### PR TITLE
[#365] Contributors profile collapse update

### PR DESCRIPTION
**Pull request by su6a12**
Add toggle button to collapse contributor profile blurb

### CODE REVIEW

**Comment by julianguyen** [comment]
I recommend squashing your commits into one so that we don't have any revert commits.

File: Gemfile.lock [hunk 1]
```diff
@@ -214,7 +214,7 @@ GEM
       addressable (~> 2.3)
```
Comment (julianguyen) [inline Gemfile.lock]: Can you exclude this file from this PR?

File: app/assets/javascripts/contributors_readmore.js [hunk 2]
```diff
@@ -15,23 +30,26 @@ var readMoreShowContent = function(){
...
-		profileTextLength = $(this).text().length;
+  contributorProfile.each(function() {
+    profileTextLength = $(this).text().length;
     if (profileTextLength > maxProfileLength){
       visibleProfileText = $(this).text().substr(0, maxProfileLength);
       invisibleProfileText = $(this).text().substr(maxProfileLength, profileTextLength - maxProfileLength);
 
-      var profileContent = visibleProfileText + '<span class="read_more_show hide_content">'+ I18n.t('ellipsis') +'</span><span class="read_more_content">' + invisibleProfileText + '</span>';
+      // Add [Less] collapse button to profile blurb
+      var profileContent = visibleProfileText + '<span class="read_more_show hide_content">'+ I18n.t('ellipsis') +'</span><span class="read_more_content">' + invisibleProfileText + '</span><span class="read_less_show"> [Less]</span>';
```
Comment (su6a12) [inline app/assets/javascripts/contributors_readmore.js]: Added **less: " [Less]"** under "en"

Comment (julianguyen) [inline app/assets/javascripts/contributors_readmore.js]: I don't see en.yml updated in this PR :P

File: Gemfile.lock (hunk 1 above)
Comment (julianguyen) [inline Gemfile.lock]: Whoops, I meant the whole `Gemfile.lock` file.

Comment (su6a12) [inline Gemfile.lock]: Do I add it to .gitignore file?

Gemfile.lock

**Comment by julianguyen** [comment]
Nope we want this file in the repo. It provides all the required gems and
their dependencies.

File: Gemfile.lock (hunk 1 above)
Comment (su6a12) [inline Gemfile.lock]: How do I exclude a file from a pull request?

Comment (julianguyen) [inline Gemfile.lock]: You'll have to unstage that file from your commit and check it out

Comment (su6a12) [inline Gemfile.lock]: Just to confirm, the command to unstage a file is **git reset <filepath>**? Would the filepath in this case be /Gemfile.lock?

**Comment by julianguyen** [comment]
Awesome work @su6a12! :D Please add yourself to the contirbutor page too in another PR :)

**Pull request merged**